src/query_parser/parser.y

```diff
@@ -837,10 +937,10 @@ vector_query(A) ::= vector_command(B). {
 
 as ::= AS_T.
 as ::= AS_S.
```

Comment:
Thanks to `%fallback` directive,
Any `as` appearing in the query, other than in a `vector_query`,
Will either be considered as a term, if `as` is not a stop-word,
Or be considered as a stop-word if it is a stop-word,
So no breaking change with `as` now becoming a keyword in lexer?

Reply:
Exactly. I’ll add a comment explaining the fallback